pkg/container/network.go

```diff
@@ -28,17 +35,31 @@ ip link set vm0 master br0
 
 // Array of container interfaces to ignore (not forward to vm)
 var ignoreInterfaces = map[string]struct{}{
-	"lo": {},
+	"lo":    {},
+	"sit0":  {},
+	"tunl0": {},
 }
 
-func SetupContainerNetworking() ([]DHCPInterface, error) {
+var (
+	mainInterface = "eth0"
+)
+
+func SetupContainerNetworking(vm *api.VM) (firecracker.NetworkInterfaces, []DHCPInterface, error) {
 	var dhcpIfaces []DHCPInterface
+	var fcIfaces firecracker.NetworkInterfaces
+
+	extraIntfs := parseExtraIntfs(vm)
+
+	// total number of interfaces is at least extraIntfs + eth0
+	totalIntfNum := len(extraIntfs) + 1
+
 	interval := 1 * time.Second
-	timeout := 1 * time.Minute
+	timeout := 2 * time.Minute
```

Comment:
Just noting that the default network-setup retry-timeout has been increased. :+1:

Reply:
yep, found that my old i5 was struggling with 2 VMs in 1 minute